_security-analytics/sec-analytics-config/correlation-config.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Creating correlation rules
+parent: Setting up Security Analytics
+nav_order: 16
+---
+
+# Creating correlation rules
+
+The correlation engine is an experimental feature released in OpenSearch 2.7. Therefore, we do not recommend using the feature in a production environment at this time. For updates on the progress of correlation engine, see [Security Analytics Correlation Engine](https://github.com/opensearch-project/security-analytics/issues/369) at GitHub. To share ideas and provide feedback, join the [Security Analytics forum](https://forum.opensearch.org/c/plugins/security-analytics/73).
+{: .warning }
+
+Correlation rules allow you to define threat scenarios involving multiple systems in an infrastructure by matching the signatures of threat events occuring in different log types. Once a rule contains at least two different log sources and the preferred fields and field values that define an intended threat secenario, the correlation engine can query the indexes specified in the correlation rule and draw any correlations found between the findings.
+
+
+## Configuring rules
+
+Follow the steps in this section to create a correlation rule.
+
+1. Begin by selecting **Security Analytics** in the Dashboards home page menu. Then select **Correlation rules** from the Security Analytics menu on the left side of the screen. The **Correlation rules** page is displayed.
+   <img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/create-corr-rule.png" alt="The correlation rules page" width="85%">
+1. Select **Create correlation rule**. The **Create correlation rule** window opens.
+1. In the **Correlation rule details** field, enter a name for the rule.
+1. In the **Correlation queries** field, specify a data source and select the index field and field values to be used for the query.
+   A minimum of two queries is required for each correlation rule. Having at least two data sources is the basis for making connections between different systems in an infrastructure and identifying correlations. You can also include more than two queries to better define a threat scenario and look for correlations between multiple systems.
```

Comment:
Could this overview information be better suited for the intro of this section? Knowing ahead of time that the correlation rule requires at least two queries to be defined may help simplify this step.